pkgs/desktops/plasma-5/plasma-desktop/default.nix

```diff
@@ -115,10 +118,20 @@ mkDerivation {
   patches = [
     ./hwclock-path.patch
     ./tzdir.patch
+    ./kcm-access.patch
   ];
-  CXXFLAGS = [
-    ''-DNIXPKGS_HWCLOCK=\"${lib.getBin util-linux}/sbin/hwclock\"''
-  ];
+  CXXFLAGS =
+    let
+      # run gsettings with desktop schemas for using in kcm_accces kcm
+      gsettings-wrapper = writeShellScript "gsettings-wrapper" ''
+        export XDG_DATA_DIRS=$XDG_DATA_DIRS:${gsettings-desktop-schemas.out}/share/gsettings-schemas/${gsettings-desktop-schemas.name}
+        exec ${glib}/bin/gsettings "$@"
+      '';
```

Comment:
We should do this with makeWrapper which is more sophisticated to match dangling ``:`` in PATH like variables

Reply:
fixed it